files/zh-cn/web/css/shorthand_properties/index.md

```diff
@@ -12,33 +12,54 @@ CSS 规范定义简写属性的目的在于将那些关于同一主题的常见
 
 虽然它们使用起来非常方便，但在使用时，仍需牢记一些边界情况：
 
-1. 没有指定的值会被设置为它的初始值。这听起来似乎本来就很合理的样子，但这确实意味着，它将会覆盖之前设置的值。因此：
+### 省略属性
 
-    ```css
-    background-color: red;
-    background: url(images/bg.gif) no-repeat top right;
-    ```
+没有指定的值会被设置为它的初始值。这听起来似乎本来就很合理的样子，但这确实意味着，它将会覆盖之前设置的值。因此：
 
-    以上样式不会将 background 的 color 值设置为 `red`，而是 {{cssxref("background-color")}} 的默认值 `transparent`，因为第二条规则优先。
+```css
+p {
+  background-color: red;
+  background: url(images/bg.gif) no-repeat left top;
+}
+```
+
+以上样式不会将 background 的 color 值设置为 `red`，而是 {{cssxref("background-color")}} 的默认值 `transparent`。
+
+关键词 inherit 只可以应用于单独属性（individual properties），如果应用于一个简写属性（shorthand property），则必须整体应用，而能对简写属性值的每一个部分单独应用。由于单独属性的漏掉的值会被它们的初始值（initial value）替代，因此不可能允许单个属性通过省略继承的。这意味着让一个属性的值使用继承值的唯一方法就是使用值是 `inherit` 的普通属性（longhand property）。
+
+### 属性的顺序
+
+简写属性不试图强制它们替代属性的值的特定顺序。这适用于当这些属性使用不同类型的值时，因为这个时候顺序并不重要。但当几个属性可以设置相同值的时候，就没那么简单了。
+
+两种非常重要的情况如下：
+
+- 与盒子（box）边界（edge）相关的属性，比如 {{cssxref("border-style")}}、{{cssxref("margin")}} 或者 {{cssxref("padding")}}
+- 与盒子的角（corner）相关的属性，比如 {{cssxref("border-radius")}}
+
+#### 盒子边界
 
-2. 关键词 inherit 只可以应用于单独属性（individual properties），如果应用于一个简写属性（shorthand property），则必须整体应用，而能对简写属性值的每一个部分单独应用。由于单独属性的漏掉的值会被它们的初始值（initial value）替代，因此不可能允许单个属性通过省略继承的。这意味着让一个属性的值使用继承值的唯一方法就是使用值是 inherit 的普通属性（longhand property）。
-3. 简写属性不试图强制它们替代属性的值的特定顺序。这适用于当这些属性使用不同类型的值时，因为这个时候顺序并不重要。但当几个属性可以设置相同值的时候，就没那么简单了。处理这些情况分以下几种类型：
+在处理和盒子的边界相关的属性时，比如 {{cssxref("border-style")}}、{{cssxref("margin")}} 或者 {{cssxref("padding")}}，始终使用一致的 1 个到 4 个值的语法表示：
 
-    1. 处理和盒子（box）边界（edge）相关的属性时，比如 {{cssxref("border-style")}}、{{cssxref("margin")}} 或者 {{cssxref("padding")}}， 始终使用一致的 1 个到 4 个值的语法表示这些边界：
+- **单值语法：** `border-width: 1em`，这一个值表示所有的边框宽度：![Box edges with one-value syntax](border1.png)
```

Comment:
Let's use this case (to avoid using an extra space):

```suggestion
- **单值语法**：`border-width: 1em`，这一个值表示所有的边框宽度：![Box edges with one-value syntax](border1.png)
```